genre.rb

```diff
@@ -0,0 +1,14 @@
+class Genre
+  attr_accessor :items
```

Comment:
- Please add an attribute reader for the name so that you can be able to access the name outside the class